<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTip</class>
 <widget class="QDialog" name="DialogTip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>提示</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/prefix1/res/monitor.png</normaloff>:/new/prefix1/res/monitor.png</iconset>
  </property>
  <widget class="QLabel" name="lbTip">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>提示信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/new/prefix1/image/info.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
